build/dsc.rake

```diff
@@ -224,8 +224,6 @@ eod
     task :build, [:module_path] do |t, args|
       module_path = args[:module_path] || default_dsc_module_path
       m = Dsc::Manager.new
-      wait_for_resources = Dir["#{module_path}/**/MSFT_WaitFor*"]
-      fail "MSFT_WaitFor* resources found - aborting type building! Please remove the following MSFT_WaitFor* DSC Resources and run the build again.\n\n#{wait_for_resources}\n" if !wait_for_resources.empty?
```

Comment:
The code is removed here, but this commit doesn't include adding the warning to the README. I don't see the warning in the README when looking at master or the rest of this PR.

Reply:
Addressed in readme